blocks/edit/da-assets/da-assets.js

```diff
@@ -27,7 +32,29 @@ async function fetchConf(path) {
 async function fetchValue(path, key) {
   if (CONFS[path]?.[key]) return CONFS[path][key];
 
-  const data = await fetchConf(path);
+  const [repoConf, orgConf] = await Promise.all([
+    fetchConf(`/${owner}/${repo}/`),
+    fetchConf(`/${owner}/`),
+  ]);
+
+  const data = null;
+
+  // Start with organization-level config
+  if (orgConf) {
+    data = {};
+    orgConf.forEach((item) => {
+      data[item.key] = item.value;
+    });
+  }
+
+  // Override with repository-level config
+  if (repoConf) {
+    if (data === null) data = {};
+    repoConf.forEach((item) => {
+      data[item.key] = item.value;
+    });
+  }
+
   if (!data) return null;
 
   const confKey = data.find((conf) => conf.key === key);
```

Comment:
`data` was an array previously, and is now an object which means that the `data.find` will fail on line 60.  Now that it's an object, you can just do a lookup

```suggestion
  const data = {};

  // Start with organization-level config
  orgConf?.forEach((item) => {
    data[item.key] = item.value;
  });

  // Override with repository-level config
  repoConf?.forEach((item) => {
    data[item.key] = item.value;
  });

  return data[key];
```

Reply:
I will be updating the PR to treat data as an array to retain the expected data type of `data` variable. Pushing objects to an array and updating values inside them if the keys match afterwards, is my approach.